rules-tests/Php83/Rector/ClassConst/AddTypeToConstRector/Fixture/skip_when_parent_not_autoloaded.php.inc

```diff
@@ -1,11 +0,0 @@
-<?php
-
-namespace Rector\Tests\Php83\Rector\ClassConst\AddTypeToConstRector\Fixture;
-
-final class SkipExtendsNotAutoloadedClass extends NotAutoloadedClass
-{
-    public const STRING = 'some_type';
-}
-
-?>
```

Comment:
why this fixture removed, there is ClassReflectionAnalyzer to verify parent class autoloaded if we want 

https://github.com/rectorphp/rector-src/blob/befe096ca10f7476efeb082fb7c736ba7bd07af5/src/Reflection/ClassReflectionAnalyzer.php#L10

Default ClassReflection has -> get can't verify that

Reply:
We should check any code without autoloaded content, as that would disable PHPStan ability.
iIt's up to user to provided correctly loaded classes. Same as that Wordpress issue, where incorrect autoloading disabled removing args.